ui/src/main/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodPresenter.java

```diff
@@ -15,23 +15,24 @@
  * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
  * IN THE SOFTWARE.
  */
-
 package com.hyperwallet.android.ui.transfermethod;
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
 import com.hyperwallet.android.model.HyperwalletErrors;
 import com.hyperwallet.android.model.HyperwalletUser;
```

Comment:
This class has a few inconsistencies with regards to variable names. In some methods, we have String country and others String countryCode. Could you please make the variable names consistent in the whole class. Same applies to other variables